### PR TITLE
bitwarden.com fixes

### PR DESCRIPTION
**Pull request by ZOASR**

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -1739,6 +1739,19 @@ CSS
 
 ================================
 
+bitwarden.com
+
+INVERT
+img[alt="Github Logo"]
+body > div > section:nth-child(4) > div > div:nth-child(1) > div > blockquote > footer > cite > img
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:1746]: What about to simplify it?
```suggestion
img[src$="forbes.png"]
```

File: src/config/dynamic-theme-fixes.config [hunk 2]
```diff
@@ -1739,6 +1739,19 @@ CSS
...
+
+CSS
+.blockquote-header{
+     background-image: url(../images/quote-marks.png) !important;
+}
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:1751]:
```suggestion
IGNORE IMAGE ANALYSIS
blockquote .blockquote-header
```

**Review by Gusted — CHANGES_REQUESTED** [review]

File: src/config/dynamic-theme-fixes.config (hunk 2 above)
Comment (ZOASR) [reply src/config/dynamic-theme-fixes.config:1751]: I tried `IGNORE IMAGE ANALYSIS` didn't work for me so i resorted to just CSS.

Comment (Gusted) [reply src/config/dynamic-theme-fixes.config:1751]: You have to apply it and then give the website a hard refresh.

Comment (ZOASR) [reply src/config/dynamic-theme-fixes.config:1751]: > You have to apply it and then give the website a hard refresh.

I did but didn't work either thats' why i got confused

Comment (Gusted) [reply src/config/dynamic-theme-fixes.config:1751]: image
does seem to work for me.

Comment (ZOASR) [reply src/config/dynamic-theme-fixes.config:1751]: Ok now i know why:
i was using refrencing only the class in the selector: `.blockquote-header`
but that should work too right?

Comment (Gusted) [reply src/config/dynamic-theme-fixes.config]: Not really, the current implementation requires the exact selector.

Comment (ZOASR) [reply src/config/dynamic-theme-fixes.config]: > Not really, the current implementation requires the exact selector.

Ok, got it. 
Thanks!.

**Review by Gusted — APPROVED** [review]

**Pull request merged**